src/sys/uio.rs

```diff
@@ -56,6 +59,38 @@ pub fn pread(fd: RawFd, buf: &mut [u8], offset: off_t) -> Result<usize>{
     Errno::result(res).map(|r| r as usize)
 }
 
+// process_vm_{read,write}v currently doesn't use flags, but we define this
+// type for forwards-compatibility.
+#[cfg(any(target_os = "linux"))]
+bitflags!(
+    pub struct CmaFlags: libc::c_ulong {
+        // The bitflags! macro requires at least one variant. any() evaluates
+        // to false, so this symbol never actually exists.
+        #[cfg(any())]
+        const CMA_DUMMY = 0;
+    }
+);
+
+#[cfg(any(target_os = "linux"))]
+pub fn process_vm_writev(pid: Pid, local_iov: &[IoVec<&[u8]>], remote_iov: &mut [IoVec<&mut [u8]>], flags: CmaFlags) -> Result<usize> {
```

Comment:
These needs docs. See some of the recently closed PRs for an idea of what we're looking for. I think these are Linux-specific functions, so you should link to a manpage on man7.org in the doc comment. Please run `cargo doc --no-deps --open` and read through the generated docs to make sure they look nice as part of your testing.